flink-core/src/main/java/org/apache/flink/api/common/state/StateTtlConfig.java

```diff
@@ -32,6 +32,12 @@
 
 /**
  * Configuration of state TTL logic.
+ *
+ * <p>Note: The map state with TTL currently supports null user values
```

Comment:
Do you think this is the best position for this comment or rather on `MapStateDescriptor` (where the serializer is supplied) or maybe even both?